src/localisation-service.ts

```diff
@@ -1,55 +1,163 @@
 import tenants from './tenants'
 
-export class LocalisationService {
```

Comment:
Why change this?

Reply:
The class in the docs looks yuck and is difficult to understand.